BaaServiceKitTests/Resources/ChainpontResponses/FakeProofResponse2.json

```diff
@@ -1,13 +0,0 @@
-// Examples
-// 1. None
-//[
```

Comment:
Why did you delete this?

Reply:
it was not used. should I keep it? The swagger should have it anyway.